UT4MasterServer.Services/Scoped/RatingsService.cs

```diff
@@ -0,0 +1,210 @@
+﻿using MongoDB.Driver;
+using UT4MasterServer.Common;
+using UT4MasterServer.Models;
+using UT4MasterServer.Models.DTO.Response;
+using UT4MasterServer.Models.DTO.Responses;
+
+namespace UT4MasterServer.Services;
```

Comment:
Use right namespace based on file location
```suggestion
namespace UT4MasterServer.Services.Scoped;
```